core/src/test/java/org/infinispan/distribution/DistSyncFuncTest.java

```diff
@@ -352,10 +352,11 @@ public void testComputeIfPresentFromNonOwner() throws InterruptedException {
       expectException(RemoteException.class, () -> getFirstNonOwner("k1").computeIfPresent("k1", mappingToException));
 
       // remove if after compute value is null
-      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
-      asyncWait("k1", ComputeCommand.class);
-      if (testRetVals) assertNull(retval);
-      assertRemovedOnAllCaches("k1");
+      // TODO: Fix https://issues.jboss.org/browse/ISPN-8074
+//      retval = getFirstNonOwner("k1").computeIfPresent("k1", (v1, v2) -> null);
+//      asyncWait("k1", ComputeCommand.class);
+//      if (testRetVals) assertNull(retval);
+//      assertRemovedOnAllCaches("k1");
```

Comment:
@danberindei This test fails randomly in this test but in any extending it, that's why I did it that way.
Concerning the JIRA ignoring the test and linking to the other JIRA that corrects the possible bug behind, @tristantarrant told me to do it like that (at least, this is what I understood)